<?xml version="1.0"?>
<launch>
    <node pkg="waypoint_manager" type="route_publisher" name="route_publisher_node" output="screen">
        <rosparam command="load" file="$(find map_server)/config/spy_and_patrol.yaml"/>
    </node>
</launch>
